IMAGE_RENDERING.md

```diff
@@ -26,78 +26,28 @@ To tackle this **_we look to [WebGL](https://developer.mozilla.org/en-US/docs/We
 
 ## Pitfalls of other routes
 
-- One possible solution is to use an already existing tiling service, like [mapbox](https://www.mapbox.com) for creating/hosting our data or **OpenSeaDragon** with DeepZoom (or some combination of the two). However, these services does not support high bit-depth-per-channel images. To use these then, we could pack 32 bits per channel (for high bit-depth-per-channel microscopy images) across the total 32 bits of the RGBA of a standard image file and then serve that and decode it, doing some sort of mapping on the shaders. Besides the kludge-y nature of this, we don't save anything really in terms of time as far as transporting the data is concerned, assuming we use `png` - we still need to serve the same amount of data. And, if we attempted to do this with `jpeg` format, it's not even clear how the color-space transformation, DCT, and entropy encoding will react to what is no longer color values, but rather a bit packing scheme.
+- One possible solution is to use an already existing tiling service, like [mapbox](https://www.mapbox.com) for creating/hosting our data or **OpenSeaDragon** with DeepZoom (or some combination of the two). However, these services does not support high bit-depth-per-channel images. To use these then, we could pack 32 bits per channel (for high bit-depth-per-channel microscopy images) across the total 32 bits of the RGBA of a standard image file and then serve that and decode it, doing some sort of mapping on the shaders. Besides the kludge-y nature of this, we don't save anything really in terms of time as far as transporting the data is concerned, assuming we use `png` - we still need to serve the same amount of data. And, if we attempted to do this with `jpeg` format, it's not clear (to me, a novice) how the color-space transformation, DCT, and entropy encoding will react to what is no longer color values, but rather a bit packing scheme.
 - Vector tiles from **mapbox** are another option, relying on **Protobuf**, but our data is not vector data. This raises questions of how we can get the data into that format, and, once it is in that format, why we would not just serve it ourselves over **gRPC** which appears to be faster.
-- I am dismissing [OMERO](https://www.openmicroscopy.org/omero/) out of hand because we do not want to deal with a server that does anything beyond serving files.
+- We dismiss [OMERO](https://www.openmicroscopy.org/omero/) out of hand because we do not want to deal with a server that does anything beyond serving files.
```

Comment:
This has a negative connotation, and I would suggest wording differently. Viv aims to utilize web technologies which weren't available until literally this last year. OMERO is a powerful tool but it relys on a server and our goal was to create a rederer that does everything on the client!

Reply:
You're totally right.  When I originally wrote this, it was mainly for Nils - sorry about that!